templates/index.html

```diff
@@ -11,7 +11,7 @@
     <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
             <div class="navbar-header">
-                <a class="navbar-brand" href="#">Route Documentation</a>
+                <a class="navbar-brand" href="#">Route Documentation{{#@apiVersion}} - v{{@apiVersion}}{{/@apiVersion}}</a>
```

Comment:
We're using handlebars, not mustache, I'd rather not rely on undocumented features, so use a if.